forge/test/model_demos/high_prio/nlp/pytorch/test_whisper_large_v3_turbo.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Whisper Large V3 turbo - Automatic speech recognition and speech translation
+# Model link : https://huggingface.co/openai/whisper-large-v3-turbo
+# By default, Transformers uses the sequential algorithm.
+# To enable the chunked algorithm, pass the chunk_length_s parameter to the pipeline.
+# For large-v3, a chunk length of 30-seconds is optimal. To activate batching over long audio files, pass the argument batch_size
+
+import torch
+from transformers import AutoModelForSpeechSeq2Seq, AutoProcessor, pipeline
+from datasets import load_dataset
+
+device = "cpu"  # "cuda:0" if torch.cuda.is_available() else "cpu"
+torch_dtype = torch.float32  # torch.float16 if torch.cuda.is_available() else torch.float32
+
+model_id = "openai/whisper-large-v3-turbo"
+
+model = AutoModelForSpeechSeq2Seq.from_pretrained(
+    model_id, torch_dtype=torch_dtype, low_cpu_mem_usage=True, use_safetensors=True
+)
+model.to(device)
+
+processor = AutoProcessor.from_pretrained(model_id)
+pipe = pipeline(
```

Comment:
Let's write tests without using pipelines. 

Ideally, we should load the model with checkpoints, and just run them like: `framework_model(inputs)`.

Reply:
will update in next patch